### PR TITLE
[utils-micro] Add support to load identities in micro

### PR DESCRIPTION
**Pull request by valeriocos**
This code allows to load identities from micro-mordred. This can be achieved by using the param --identities-load. An example of identities file is also included.

### CODE REVIEW

File: utils/micro.py [hunk 1]
```diff
@@ -38,14 +38,15 @@
...
-def micro_mordred(cfg_path, backend_sections, raw, arthur, identities, enrich, panels):
+def micro_mordred(cfg_path, backend_sections, raw, arthur, identities_load, identities_merge, enrich, panels):
     """Execute the raw and/or the enrich phases of a given backend section defined in a Mordred configuration file.
 
     :param cfg_path: the path of a Mordred configuration file
     :param backend_sections: the backend sections where the raw and/or enrich phases will be executed
     :param raw: if true, it activates the collection of raw data
     :param arthur: if true, it enables Arthur to collect the raw data
-    :param identities: if true, it activates the identities merge in SortingHat
+    param identities_load: if true, it activates the identities load
```
Comment (sduenas) [inline utils/micro.py]: This is not a valid entry for a comment annotation.

File: utils/micro.py [hunk 2]
```diff
@@ -180,7 +196,7 @@ def get_params():
     parser = get_params_parser()
     args = parser.parse_args()
 
-    if not args.raw and not args.enrich and not args.identities and not args.panels:
+    if not args.raw and not args.enrich and not args.identities_load and not args.identities_merge and not args.panels:
```
Comment (sduenas) [inline utils/micro.py]: This line is too long to read, please split it in several lines.

File: utils/grimoirelab_valid.yml [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+    - owlbot@bitergia.com
+  enrollments:
+    - organization: Unknown
+      start: 2017-01-01T00:00:00
+
+- blacklist:
+    - no-reply@example.com
+    - root
+    - Generic Account
+    - valcos@bitergia.com
```
Comment (sduenas) [inline utils/grimoirelab_valid.yml]: Add a blank line at the end of the file.

**Comment by valeriocos** [comment]
I have addressed your comments (plus the comments at #300). Please @sduenas review it, thanks

**Pull request merged**